Marlin/src/pins/lpc1768/pins_BTT_SKR_common.h

```diff
@@ -138,16 +138,16 @@
   #error "No custom SD drive cable defined for this board."
 #endif
 
-#if HAS_BTT_EXP_MOT
+#if ENABLED(BTT_MOTOR_EXPANSION)
 
-  /**              _____                                      _____
-   *           NC | · · | GND                             NC | · · | GND
-   *           NC | · · | 1.31 (M1EN)            (M2EN) 1.23 | · · | 1.22 (M3EN)
-   * (M1STP) 0.18 | · ·   3.25 (M1DIR)           (M1RX) 1.21 | · ·   1.20 (M1DIAG)
-   * (M2DIR) 0.16 | · · | 3.26 (M2STP)           (M2RX) 1.19 | · · | 1.18 (M2DIAG)
-   * (M3DIR) 0.15 | · · | 0.17 (M3STP)           (M3RX) 0.28 | · · | 1.30 (M3DIAG)
-   *               -----                                      -----
-   *               EXP2                                       EXP1
+  /**       _____                        _____
+   *    NC | 1 2 | GND               NC | 1 2 | GND
+   *    NC | 3 4 | M1EN            M2EN | 3 4 | M3EN
+   * M1STP | 5 6   M1DIR           M1RX | 5 6   M1DIAG
+   * M2DIR | 7 8 | M2STP           M2RX | 7 8 | M2DIAG
+   * M3DIR | 9 10| M3STP           M3RX | 9 10| M3DIAG
```

Comment:
Removing the passed-through pin numbers from the EXP pins makes this block of code easier to copy to future pins files.